cyences_app_for_splunk/default/macros.conf

```diff
@@ -682,6 +682,16 @@ iseval = 0
 definition = search *
 iseval = 0
 
+[cs_windows_cert_store_idx]
+definition = index IN (wincertstore)
+iseval = 0
+
+[cs_windows_cert_name_mapping]
+definition = ```Add each Certificate_Template and Certificate_Name pair in a new line separated by commas.```\
```

Comment:
Let's go with the lookup instead. If the lookup is not present in the customer environment it will generate the error, but search will not fail.

We can add not that you user will see "The lookup table 'hello.csv' requires a .csv or KV store lookup definition." error for this alert and this dashboard in case .... and if you can wants to configure you can do it by going to ....
* In the Cyences Doc troubleshooting section
* Under the dashboard
* Alert description